Mathlib/Analysis/CStarAlgebra/Unitization.lean

```diff
@@ -30,10 +30,13 @@ variable [NormedSpace 𝕜 E] [IsScalarTower 𝕜 E E] [SMulCommClass 𝕜 E E]
 lemma opNorm_mul_flip_apply (a : E) : ‖(mul 𝕜 E).flip a‖ = ‖a‖ := by
   refine le_antisymm
     (opNorm_le_bound _ (norm_nonneg _) fun b => by simpa only [mul_comm] using norm_mul_le b a) ?_
-  suffices ‖mul 𝕜 E (star a)‖ ≤ ‖(mul 𝕜 E).flip a‖ by simpa using this
+  suffices ‖mul 𝕜 E (star a)‖ ≤ ‖(mul 𝕜 E).flip a‖ by
+    simpa only [ge_iff_le, opNorm_mul_apply, norm_star] using this
   refine opNorm_le_bound _ (norm_nonneg _) fun b => ?_
-  calc ‖mul 𝕜 E (star a) b‖ = ‖(mul 𝕜 E).flip a (star b)‖ := by simpa using norm_star (star b * a)
-    _ ≤ ‖(mul 𝕜 E).flip a‖ * ‖b‖ := by simpa using le_opNorm ((mul 𝕜 E).flip a) (star b)
+  calc ‖mul 𝕜 E (star a) b‖ = ‖(mul 𝕜 E).flip a (star b)‖ := by
+        simpa only [mul_apply', flip_apply, star_mul, star_star] using norm_star (star b * a)
+    _ ≤ ‖(mul 𝕜 E).flip a‖ * ‖b‖ := by
+        simpa only [flip_apply, mul_apply', norm_star] using le_opNorm ((mul 𝕜 E).flip a) (star b)
```

Comment:
This declaration got a *lot* slower (I believe, in part, because there is now a `Analysis.Complex.Basic` transitive import), but I had trouble tracing it down. Squeezing the `simp`s fixed it though.